### PR TITLE
fix: wellbore class [release]

### PR DESCRIPTION
**Pull request by Sollimann**

### CODE REVIEW

File: packages/wells/src/client/model/Wellbore.ts [hunk 1]
```diff
@@ -48,12 +48,6 @@ export interface Wellbore {
    * @memberof Wellbore
    */
   id: number;
-  /**
-   * Wellbore description
-   * @type {string}
-   * @memberof Wellbore
-   */
-  description?: string;
```
Comment (sighol) [inline packages/wells/src/client/model/Wellbore.ts]: Why have you removed the wellbore description?

Comment (drsphelps) [inline packages/wells/src/client/model/Wellbore.ts:56]: Didn't someone add the description for wellbores last week on request from discover?

Comment (Sollimann) [reply packages/wells/src/client/model/Wellbore.ts]: Couldn't find it in the data class: 
image

Comment (drsphelps) [reply packages/wells/src/client/model/Wellbore.ts]: Make sure you have the latest version of the service, it was only added recently

**Review by drsphelps — APPROVED** [review]

**Pull request merged**